### ISSUE TITLE
Test cases appear to only work if they are in a directory called `simple`

### ISSUE THREAD

**Issue by timhughes** [kennt/fixtest#1]
Test cases appear to only work if they are in a directory called `simple`. I have experimented with different names and nothing else appears to work. I cannot find anything mentioning `simple` so I cannot work out what is causing the issue.

```
[hughest@argon: ~/git/fixmon]$ fixtest --debug -c config/testcases_config.py simple/logon_controller.py 
15:16:53.355005: ================
15:16:53.355157: Starting test: 2015-04-17
15:16:53.355244:   Module: simple/logon_controller.py
15:16:53.355313:   Controller: LogonController
15:16:53.355386:   Config: config/testcases_config.py
15:16:53.355806: 
15:16:53.355882:   Test case: Simple-1
15:16:53.355934:   Description: Test of the command-line tool
15:16:53.355999: ================
15:16:53.356064: server:server-9940 starting on port 9940
FIXTransportFactory starting on 9940
Starting factory <fixtest.fix.transport.FIXTransportFactory instance at 0x174fd88>
15:16:53.356695: fixtest.fix.transport: server:server-9940 listening on port 9940
15:16:53.357097: client:client-9940 attempting localhost:9940
Starting factory <twisted.internet.endpoints.OneShotFactory instance at 0x1996200>
15:16:53.375914: Connected: fixtest.fix.transport.FIXTransportFactory : server-9940
15:16:53.376359: server-9940: Connection made
15:16:53.376680: client-9940: Connection made
15:16:53.376766: fixtest.fix.transport: client:client-9940 connected to localhost:9940
15:16:53.557964: client-9940: message sent
    Logon : 8=FIX.4.2, 9=68, 35=A, 49=FixClient, 56=FixServer, 98=0, 108=5, 34=1, 52=20150417-15:16:53, 10=051

15:16:53.558954: server-9940: message received
    Logon : 8=FIX.4.2, 9=68, 35=A, 49=FixClient, 56=FixServer, 98=0, 108=5, 34=1, 52=20150417-15:16:53, 10=051

15:16:53.560020: server-9940: message sent
    Logon : 8=FIX.4.2, 9=68, 35=A, 49=FixServer, 56=FixClient, 98=0, 108=5, 34=1, 52=20150417-15:16:53, 10=051

15:16:53.560913: client-9940: message received
    Logon : 8=FIX.4.2, 9=68, 35=A, 49=FixServer, 56=FixClient, 98=0, 108=5, 34=1, 52=20150417-15:16:53, 10=051

15:16:53.561929: client-9940: message sent
    Logout : 8=FIX.4.2, 9=57, 35=5, 49=FixClient, 56=FixServer, 34=2, 52=20150417-15:16:53, 10=059

15:16:53.562708: server-9940: message received
    Logout : 8=FIX.4.2, 9=57, 35=5, 49=FixClient, 56=FixServer, 34=2, 52=20150417-15:16:53, 10=059

15:16:53.563788: server-9940: message sent
    Logout : 8=FIX.4.2, 9=57, 35=5, 49=FixServer, 56=FixClient, 34=2, 52=20150417-15:16:53, 10=059

15:16:53.564416: client-9940: message received
    Logout : 8=FIX.4.2, 9=57, 35=5, 49=FixServer, 56=FixClient, 34=2, 52=20150417-15:16:53, 10=059

15:16:53.565381: server-9940: Connection lost
15:16:53.565545: client-9940: Connection lost
Stopping factory <twisted.internet.endpoints.OneShotFactory instance at 0x1996200>
(TCP Port 9940 Closed)
Stopping factory <fixtest.fix.transport.FIXTransportFactory instance at 0x174fd88>
Main loop terminated.
15:16:53.566051: ================
15:16:53.566116: Test status: ok
```

```
[hughest@argon: ~/git/fixmon]$ mv simple testcases
```

```
[hughest@argon: ~/git/fixmon]$ fixtest --debug -c config/testcases_config.py testcases/logon_controller.py 
Traceback (most recent call last):
  File "/home/hughest/.virtualenvs/fixmon/bin/fixtest", line 9, in <module>
    load_entry_point('fixtest==0.1.0', 'console_scripts', 'fixtest')()
  File "/home/hughest/.virtualenvs/fixmon/lib/python2.7/site-packages/fixtest/base/runner.py", line 187, in main
    controller_class = _find_controller(arg_results.test_name)
  File "/home/hughest/.virtualenvs/fixmon/lib/python2.7/site-packages/fixtest/base/runner.py", line 106, in _find_controller
    module = importlib.import_module(module_path)
  File "/home/hughest/.pyenv/versions/2.7.5/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named testcases.logon_controller
```

**Comment by kennt** [comment]
Interesting. I remember having the same issue. But I can't remember why. Something about importing the Python modules?  I think it was the difference between running the tests with the module or having fixtest already installed. I'll look into it on Monday. 

Thanks,
Kenn

**Comment by kennt** [comment]
Found the problem.  Needed to add the current directory to the system path when searching for the TestController module.  It was working for simple since simple is part of fixtest itself.

Updated and released as version 0.1.1
